tests/README.md

```diff
@@ -13,6 +13,11 @@ Note: Not all tests have been converted to tox and/or pytest yet.
 
 ## Running the tests
 
+Before running storage tests, make sure you have setup the user storage
+accordingly. Read
+[Setting up user storage](./storage/README.md#setting-up-user-storage)
+for more information.
```

Comment:
It is better to show how to setup the storage here:

    make storage

And link the storage readme for more info.